ed448/src/curve/twedwards/extended.rs

```diff
@@ -143,14 +143,13 @@ impl ExtendedPoint {
         let XY = self.X * self.Y;
         let ZT = self.Z * self.T;
 
-        // Y^2 - X^2 == Z^2 + T^2 * (TWISTED_D)
```

Comment:
I couldn't find this formula anywhere, but I guess its just the *twisted* edwards curve?
The non-twisted one is: `x² + y² = 1 + dx² y²`.

This seems to be a general theme here: The current implementation uses twisted, but for me to get it working with the VOPRF test vectors I have to use an untwisted implementation.